src/aind_data_access_api/util/data_schema.py

```diff
@@ -0,0 +1,62 @@
+"""Module for convenience functions for the data access API."""
+
+from typing import Optional
+from aind_data_access_api.document_db import MetadataDbClient
+from aind_data_access_api.util.docdb import (
+    get_projected_record_from_docdb,
+    get_id_from_name,
+)
+from aind_data_schema.core.quality_control import QualityControl
+import json
+
+
+def get_quality_control(
+    client: MetadataDbClient,
+    _id: Optional[str] = None,
+    name: Optional[str] = None,
+    allow_invalid: bool = False,
+):
+    """Using a connected DocumentDB client, retrieve the QualityControl object
+    for a given record.
+
+    Parameters
+    ----------
+    client : MetadataDbClient
+        A connected DocumentDB client.
+    id : str, optional
+        _id field in DocDB, by default empty
+    name : str, optional
+        name field in DocDB, by default empty
+    allow_invalid : bool, optional
+        return invalid QualityControl as dict if True, by default False
+    """
+    if not _id and not name:
+        raise ValueError("Must specify either _id or name.")
+
+    if name:
```

Comment:
It looks like `name` is used over `_id` if both are provided. This might be confusing to users. We should either document this behavior in the docstring, or consider separating into 2 methods: e.g. `get_quality_control_by_name` and `get_quality_control_by_id`